### PR TITLE
Prac 05  review request

### PR DESCRIPTION
**Pull request by ChristineYamin**
@TinNilarSoe
@ivy-naw-mth

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+"""
+CP1404/CP5632 Practical
+hex colours in a dictionary
+"""
+
+CODE_TO_NAME={"Absolute Zero": "#0048ba","Acid Green" : "#b0bf1a", "Alice Blue":"#f0f8ff","Baby Pink" : "#f4c2c2","Barn Red" : "#7c0a02","Beaver" : "#9f8170", "Bitter Lime" : "#bfff00","Blond" : "#faf0be","buff" : "#f0dc82","Camel" : "#c19a6b"}
+
```
Comment (TinNilarSoe) [inline prac_05/hex_colours.py:8]: "buff" should be "Buff" ^^

File: prac_05/wimbledon.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    records = []
+    with open(filename, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()  # Remove header
+        for line in in_file:
+            parts = line.strip().split(",")
+            records.append(parts)
+    return records
+
+
+main()
```
Comment (TinNilarSoe) [inline prac_05/wimbledon.py:52]: Good code!

File: prac_05/hex_colours.py (hunk 1 above)
Comment (ChristineYamin) [reply prac_05/hex_colours.py:8]: I will fix that .
Thanks you.

**Comment by Anastasia-Eremenko** [comment]
Good job for completing the practical
Your code structure is quite clear with proper exercise headings

File: prac_05/emails.py [hunk 3]
```diff
@@ -13,7 +13,7 @@ def main():
         username = get_name(email)
         enquiry = input(f"Is your name{username}? (Y/n)").upper()
         if enquiry != "Y" and enquiry != "":
-            real_name = input("Name :")
```
Comment (Anastasia-Eremenko) [inline prac_05/emails.py]: The naming convention between real_name and username can be confusing
but it works for this exercise

Comment (ChristineYamin) [reply prac_05/emails.py]: I will fix that 
Thanks for your feedback

File: prac_05/emails.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+def get_name(email):
+    """ Get name from the email """
+    fullname= email.split("@")[0]
+    name=fullname.split(".")
+    return " ".join(name).title()
+main()
+
+
+
```
Comment (TinNilarSoe) [inline prac_05/emails.py:30]: great code!

File: prac_05/hex_colours.py (hunk 1 above)
Comment (ivy-naw-mth) [reply prac_05/hex_colours.py:8]: Changes applied to the comment, great job! ^^

**Pull request merged**